engine/enginerpc/engine_impl.cc

```diff
@@ -941,6 +941,47 @@ grpc::Status engine_impl::GetContactGroup(grpc::ServerContext* context
     return grpc::Status(grpc::INVALID_ARGUMENT, err);
 }
 
+/**
+ * @brief Return Tag informations.
+ *
+ * @param context gRPC context
+ * @param request Tag's identifier (by Tag id and type)
+ * @param response The filled fields
+ *
+ *@return Status::OK
```

Comment:
I see in the code that we can send others statuses

Reply:
done